content/en/tracing/trace_collection/admission_controller.md

```diff
@@ -31,25 +36,37 @@ Adding this annotation results in the injection of the tracer library for that l
 Valid `<language>` values are:
 - `java`
 - `js`
+- `python`
 
-For example to inject the latest Java tracer:
+For example to inject the Java tracer v1.0.0:
 
 ```yaml
 annotations:
-    admission.datadoghq.com/java-lib.version: "latest"
+    admission.datadoghq.com/java-lib.version: "1.0.0"
```

Comment:
I don't think v1.0.0 is available for java in this mode ( at least for [now](https://hub.docker.com/repository/registry-1.docker.io/datadog/dd-lib-java-init/tags?page=1&ordering=last_updated)).